src/assets/scripts/client/App.js

```diff
@@ -66,27 +43,18 @@ export default class App {
          * @default body
          */
         this.$element = $(element);
+        this._appController = new AppController(this.$element);
         this.eventBus = EventBus;
-        this.loadingView = null;
-        this.contentQueue = null;
-        this.airlineCollection = null;
-        this.airportController = null;
-        this.tutorialView = null;
-        this.aircraftCommander = null;
-        this.inputController = null;
-        this.uiController = null;
-        this.canvasController = null;
 
         window.prop = prop;
 
         this.prop = prop;
         this.prop.complete = false;
-        this.prop.temp = 'nothing here';
         this.prop.version = VERSION;
         this.prop.time = {};
         this.prop.time.start = time();
         this.prop.time.frames = 0;
-        this.prop.time.frame = {};
+        // this.prop.time.frame = {};
         this.prop.time.frame.start = time();
```

Comment:
yes you would.  this is very much a WIP, I just wanted to push it and open the PR to see where I'm at with develop.  and to let you take a peak, too.